### PR TITLE
Add 4.5.0 CHANGELOG

### PR DESCRIPTION
**Pull request by theopolis**
Thanks to: https://github.com/osquery/packaging-tools

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,88 @@
+<a name="4.5.0"></a>
+## [4.5.0](https://github.com/osquery/osquery/releases/tag/4.5.0)
+
+[Git Commits](https://github.com/osquery/osquery/compare/4.4.0...4.5.0)
+
+### New Features
+
+- Aarch64 beta support for Linux ([#6612](https://github.com/osquery/osquery/pull/6612))
```
Comment (directionless) [inline CHANGELOG.md:8]: Consensus from slack was that we wanted to thank the contributors here.

Comment (theopolis) [reply CHANGELOG.md:8]: My preference is to have thanks in the blog post https://github.com/osquery/osquery-site/pull/183

Can do both though.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,92 @@
...
+- Update schedule counter behavior ([#6223](https://github.com/osquery/osquery/pull/6223))
+- Define UNICODE and _UNICODE preprocessors for windows ([#6338](https://github.com/osquery/osquery/pull/6338))
+- Add WMI utility function to convert datetime to FILETIME ([#5901](https://github.com/osquery/osquery/pull/5901))
+- Move osquery shutdown logic outside of Initializer ([#6530](https://github.com/osquery/osquery/pull/6530))
+
+### Table Changes
+
+- Support for Windows Background Activity Moderator ([#6585](https://github.com/osquery/osquery/pull/6585))
+- Add apparmor_events table to Linux ([#4982](https://github.com/osquery/osquery/pull/4982))
+- Add sigurl column to get YARA signatures from and HTTPS server ([#6607](https://github.com/osquery/osquery/pull/6607))
```
Comment (zwass) [inline CHANGELOG.md:32]:
```suggestion
- Add sigurl column to get YARA signatures from an HTTPS server ([#6607](https://github.com/osquery/osquery/pull/6607))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,92 @@
...
+- Add sigrules column to pass YARA signatures within queries ([#6568](https://github.com/osquery/osquery/pull/6568))
+- Add non-evented table for querying Windows Events Log ([#6563](https://github.com/osquery/osquery/pull/6563))
```
Comment (zwass) [inline CHANGELOG.md:34]:
```suggestion
- Add non-evented table `windows_eventlog` for querying Windows Events Log ([#6563](https://github.com/osquery/osquery/pull/6563))
```

**Review by zwass — CHANGES_REQUESTED** [review]
Minor changes

**Comment by theopolis** [comment]
Thanks @zwass, just applied your suggestions.

**Review by zwass — APPROVED** [review]

**Pull request merged**